altair/vegalite/v5/schema/core.py

```diff
@@ -7054,6 +7054,13 @@ class IntervalSelectionConfig(VegaLiteSchema):
         An array of encoding channels. The corresponding data field values must match for a
         data tuple to fall within the selection.
 
+        **See also:** The `projection with encodings and fields section
+        <https://vega.github.io/vega-lite/docs/selection.html#project>`__ in the
+        documentation.
+    fields : List(:class:`FieldName`)
+        An array of field names whose values must match for a data tuple to fall within the
+        selection.
+
```

Comment:
As mentioned in https://github.com/vega/vega-lite/pull/9066#issuecomment-1741211330, this is incorrect and will likely be removed in 5.16